README.md

```diff
@@ -5,7 +5,7 @@
 ### Installing
 
 ```bash
-go get gopkg.in/DataDog/dd-trace-go.v1/...
+go get gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer
```

Comment:
This command will only install the tracer for manual instrumentation, whereas this repository also has a bunch of integrations, an Opentracing wrapper, and a profiler. It seems to only cover a sub-section.

Given that this is a README at the root, we should maybe elaborate for all the above cases? Otherwise I feel we are giving incomplete instructions...

Reply:
Maybe we should have 4 sections:
* Installing the Go tracer (ddtrace/tracer)
* Installing our Opentracing wrapper (ddtrace/opentracer)
* Installing an integration (contrib/<integration_path>)
* Installing the profiler (profiler)

What do you think?

Reply:
@gbbr I played with it a bit more. For Go <= 1.15 it seems that installing the main package `gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer` was good enough, and editors like VS Code would fetch sub-packages automatically as needed. In Go 1.16 it seems like one has to manually run `go get ...` for every package 😞.

Anyway, based on your feedback, I tried to update the README in a way that I think will make the most sense for the average user. PTAL

Reply:
This is great Felix. Thanks a lot. Can we remove `opentracer` from the list? It's not something that's "usually" needed. It's just a wrapper for Opentracing, which is now superseeded by OpenTelemetry. We can add a separate section (after the integrations), or leave it out. Either is fine by me.

Reply:
@gbbr okay, PTAL at 385f452c3f2476eb0c1dd7120f1255b312c79cf3. Also added a link to the go profiler docs.